src/playground-project.ts

```diff
@@ -147,7 +181,13 @@ export class PlaygroundProject extends LitElement {
   }
 
   render() {
-    return html`<slot @slotchange=${this._slotChange}></slot>`;
+    return html`
+      <slot @slotchange=${this._slotChange}></slot>
+      <iframe
```

Comment:
Can you add a comment somewhere on the purpose of this iframe? Do we have architecture docs anywhere?

Reply:
Added brief comment. No docs, but I was going to update the diagram and include it in the README when I get around to that.